Readme.md

```diff
@@ -1,155 +1,135 @@
-# WIP libgccjit codegen backend for rust
+# GCC codegen backend for Rust (WIP)
 
 [![Chat on IRC](https://img.shields.io/badge/irc.libera.chat-%23rustc__codegen__gcc-blue.svg)](https://web.libera.chat/#rustc_codegen_gcc)
 [![Chat on Matrix](https://img.shields.io/badge/matrix.org-%23rustc__codegen__gcc-blue.svg)](https://matrix.to/#/#rustc_codegen_gcc:matrix.org)
 
-This is a GCC codegen for rustc, which means it can be loaded by the existing rustc frontend, but benefits from GCC: more architectures are supported and GCC's optimizations are used.
-
-**Despite its name, libgccjit can be used for ahead-of-time compilation, as is used here.**
-
-## Motivation
-
-The primary goal of this project is to be able to compile Rust code on platforms unsupported by LLVM.
-A secondary goal is to check if using the gcc backend will provide any run-time speed improvement for the programs compiled using rustc.
-
-### Dependencies
-
-**rustup:** Follow the instructions on the official [website](https://www.rust-lang.org/tools/install)
-
-**DejaGnu:** Consider to install DejaGnu which is necessary for running the libgccjit test suite. [website](https://www.gnu.org/software/dejagnu/#downloading)
-
+This GCC codegen enables the official rustc compiler frontend to leverage GCC as a compiler backend. This is accomplished through interfacing with GCC's [libgccjit](https://gcc.gnu.org/wiki/JIT) library. While libgccjit is primarily intended to be used as an interface to GCC for just-in-time code generation, it can also be used for ahead-of-time compilation, as we do here.
 
+### Goals
+* Enable the compilation of Rust code for target platforms supported by GCC, but not LLVM
+* Leverage potential GCC optimizations for run-time speed improvement for Rust applications
+* Reduce dependence of the Rust ecosystem on LLVM
 
 ## Building
+### Dependencies
+* **[rustup](https://www.rust-lang.org/tools/install)**: The rustup tool is required to build this project using the following instructions; do not rely on a Rust toolchain that may have been provided by your operating system's package manager.
+* **[DejaGnu](https://www.gnu.org/software/dejagnu/#downloading)** (optional): Install the DejaGnu testing framework in order to run the libgccjit test suite.
```

Comment:
I didn't install it and yet tests are working fine for me?

Reply:
If you're talking about DejaGnu, it is required and I added this dependency in a previous PR because some people did not have this installed.